reservations/notifications.py

```diff
@@ -0,0 +1,8 @@
+from django_ilmoitin.registry import notifications
+
+notifications.register("event_created", "Event created")
```

Comment:
I think it would be best to keep the enums for the codes, like
```python
class NotificationType(Enum):
    BERTH_RESERVATION_CREATED = "berth_reservation_created"

notifications.register(NotificationType.BERTH_RESERVATION_CREATED, "Berth reservation created")
```

Reply:
also, where do these `"event_..."` come from? 🙂 don't think we had those in this project before

Reply:
also, label string (e.g. `"Berth reservation created"`) should probably be translatable?